_data/current.yml

```diff
@@ -72,6 +72,14 @@
   cfp_phrase: CFP closes
   cfp_date: "October 26, 2017"
 
+- name: RubyConf Philippines
+  location: Manila, Philippines
+  dates: "March 15 - 17, 2018"
+  url: http://rubyconf.ph/
+  twitter: rubyconfph
+  cfp_phrase: CFP is open
```

Comment:
This should be "CFP closes" because with the `cfp_date` key, then the site will end up reading like this:

```
CFP closes in a month
```

or whatever.